src/services/cronJobs/checkAndUpdateEndaomentProject.ts

```diff
@@ -0,0 +1,79 @@
+import axios from 'axios';
+import { schedule } from 'node-cron';
+import { Project, ProjStatus } from '../../entities/project';
+import { logger } from '../../utils/logger';
+
+const cronJobTime = '0 0 1 * *'; // Runs once a month
```

Comment:
Set this as a env var in case we have to modify it, so no need to redeploy.
And leave this as the default value in case the env var isnt set.

Reply:
thx, great suggestion